instructions to compile on windows do not work

Greetings!

I have followed instructions to compile on windows 10.  I did not perform steps for CUDA 8.0+ nor did I download and create the directory structure for the dependency folder - again, no CUDA needed; I have a Vega.  When I create the build directory and cd to it and then run the cmake command, I get `Error: could not load cache`

Why am I changing to an empty directory and running cmake without specifying the location of make files or whatnot?  Anyway, I would like this to work, please.  :)  Please look at your instructions and let me know how I screwed up.  Thanks!

So I went through it again.  This time I grabbed the cuda stuff, assuming cuda 9, despite not having nvidia.  Got down to the last cmake command in windows compile instructions and then this:
`C:\xmr-stak-amd-final\build>cmake --build . --config Release --target install
Microsoft (R) Build Engine version 15.4.8.50001 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1009: Project file does not exist.
Switch: install.vcxproj

C:\xmr-stak-amd-final\build>`

--- a/.appveyor.yml
+++ b/.appveyor.yml
@@ -29,4 +29,4 @@ test_script:
   - cd c:\xmr-stak\build\bin\Release
   - dir
   - copy C:\xmr-stak-dep\openssl\bin\* .
-  - xmr-stak.exe --help
+#  - xmr-stak.exe --help

--- a/doc/compile_Windows.md
+++ b/doc/compile_Windows.md
@@ -81,15 +81,19 @@
   mkdir build
   cd build
   ```
-  - with CUDA 8
+  - for CUDA 8*
     ```
     cmake -G "Visual Studio 15 2017 Win64" -T v140,host=x64 ..
     ```
-  - with CUDA 9
+  - for CUDA 9 and/or AMD GPUs, CPU
     ```
     cmake -G "Visual Studio 15 2017 Win64" -T v141,host=x64 ..
     ```
   ```
   cmake --build . --config Release --target install
   cd bin\Release
+  copy C:\xmr-stak-dep\openssl\bin\* .
   ```
+
+\* Miner is also compiled for AMD GPUs (if the AMD APP SDK is installed) and CPUs.
+CUDA 8 requires a downgrade to the old v140 tool chain.

